src/commands.rs

```diff
@@ -147,11 +151,14 @@ impl<'a, V: SelectValue> KeyValue<'a, V> {
     }
 
     fn get_values<'b>(&'a self, path: &'b str) -> Result<Vec<&'a V>, Error> {
-        let mut selector = Selector::new();
-        selector.str_path(path)?;
-        selector.value(self.val);
-        let results = selector.select()?;
+        let query = compile(path)?;
+        let results = calc_once(query, self.val);
         Ok(results)
+        // let mut selector = Selector::new();
```

Comment:
Can drop commented-out code (also in other locations)

Reply:
Dropped